infrastructure/README.md

```diff
@@ -69,4 +115,6 @@ cdk deploy KITInfrastructure --no-rollback \
 | TestServiceAccount  | Service Account for application tests to run with                                          |                                                         |   |   |
 | KITAddon            | KIT CRD addon that gets installed on KIT Infrastructure by default                         | true                                                    |   |   |                               
 | KarpenterAddon      | Karpenter CRD addon that gets installed on KIT Infrastructure by default                   | true                                                    |   |   | 
-| AWSEBSCSIDriverAddon| AWSEBSCSIDriver CRD addon that gets installed on KIT Infrastructure by default             | true                                                    |   |   |
\ No newline at end of file
+| AWSEBSCSIDriverAddon| AWSEBSCSIDriver CRD addon that gets installed on KIT Infrastructure by default             | true                                                    |   |   |
+
+[perf dash]: https://github.com/kubernetes/perf-tests/tree/master/perfdash
```

Comment:
Why different formatting?

Reply:
Answered in https://github.com/awslabs/kubernetes-iteration-toolkit/pull/306#discussion_r1009719638